src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -20,7 +20,14 @@ Item {
   property ViewStatus __viewStatus
 
   //on all mimetypes image/... and on empty values it should appear as an image widget
-  property bool isImage: value === undefined || FileUtils.mimeTypeName( qgisProject.homePath + '/' + value ).startsWith("image/") || FileUtils.fileName( qgisProject.homePath + '/' + value ) === ''
+  property bool isImage: value === undefined
+    || (
+      ! config.UseLink
```

Comment:
This is the new part, correct?
Could you adjust line 22 to reflect this?

Reply:
It is still valid, no need to change the comment.